main.go

```diff
@@ -194,7 +193,7 @@ func main() {
 
 	// Make sure certs are generated and valid if cert rotation is enabled.
 	setupFinished := make(chan struct{})
-	if !*disableCertRotation {
+	if !*webhook.DisableCertRotation {
```

Comment:
nit: I would leave this flag in `main.go` since that is where the value is consumed (and it looks like webhooks no longer use this value)